alpine/LandauDampingManager.h

```diff
@@ -324,59 +349,64 @@ class LandauDampingManager
 
     void gatherCIC() {
         using Base                        = ippl::ParticleBase<ippl::ParticleSpatialLayout<T, Dim>>;
-        Base::particle_position_type *E_p = &this->pcontainer_m->getE();
-        Base::particle_position_type *R_m = &this->pcontainer_m->R;
-        VField_t<T, Dim> *E_f             = &this->fcontainer_m->getE();
-        gather(*E_p, *E_f, *R_m);
+        Base::particle_position_type *Ep = &pcontainer_m->E;
```

Comment:
Here why can't you directly pass `pcontainer_m->E` and need to define these temporaries?

Reply:
Done in 94ec049